src/lib/parser.ts

```diff
@@ -118,3 +118,66 @@ const parseJcalAlarm = (component: Component): Dictionary => {
 
   return alarmValue;
 }
+
+/**
+ * Parse a freebusy reply
+ *
+ * @param {String} ics - the vcalendar string containing the vfreebusy component
+ * @returns {FreeBusy[]}
+ */
+export function parseFreeBusy(ics: string): FreeBusy[] {
+  const jcal = parseIcs(ics);
+  const vcalendarComponent = new Component(jcal);
+  const freeBusyComponents = vcalendarComponent.getAllSubcomponents('vfreebusy');
+
+  if (!freeBusyComponents) throw new Error('no busy time information were found');
+
+  return freeBusyComponents.map(parseFreeBusyComponent);
+}
+
+/**
+ * Parse a vfreebusy component
+ * @param {ICAL.Component} component
+ * @returns {FreeBusy}
+ */
+const parseFreeBusyComponent = (component: Component): FreeBusy => {
+  const props: Property[] = component.getAllProperties();
+  const freeBusyObject: Dictionary = {};
+
+  props.forEach(prop => {
+    switch (prop.name) {
+    case 'attendee':
```

Comment:
Wrong indentation.

Reply:
that conflicts with the `eslint` rules setup in the library
![image](https://user-images.githubusercontent.com/6764881/116568560-7b614b00-a900-11eb-9493-65341ca6102c.png)